serveradmin/common/templates/opensearch.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<OpenSearchDescription xmlns="http://a9.com/-/spec/opensearch/1.1/">
+<ShortName>ServerAdmin</ShortName>
+<Description>Server shell</Description>
+<Image width="16" height="16" type="image/x-icon">/favicon.ico</Image>
+<Url type="text/html" method="GET" template="{{ SCHEMA_FQDN }}/servershell/?term={searchTerms}">
+</Url>
+<Url type="application/opensearchdescription+xml" rel="self" template="{{ SCHEMA_FQDN }}/{{ STATIC_URL }}/opensearch.xml">
+</Url>
+<InputEncoding>UTF-8</InputEncoding>
+</OpenSearchDescription>
```

Comment:
You need to a view to use this template.  I think `servershell` would be the proper module for it.  See `urls.py` and `views.py`.  I don't think you can make this work without testing your changes.